tests/MyApp.ApplicationIsolationTests/Tests/Commands/Auth/CleanupConfirmationsTests.cs

```diff
@@ -36,20 +38,21 @@ private async Task<int> CreateUserAndConfirmations()
     private async Task ExpireConfirmations(int userId)
```

Comment:
make this public and use it in all tests where you are arranging an expired confirmation